tests/flow/test_query_validation.py

```
@@ -320,25 +320,23 @@ def test22_procedure_validations(self):
-        #try:
-        #    query = """CALL db.idx.fulltext.queryNodes('A', 'B') YIELD node AS n RETURN node"""
-        #    self.graph.query(query)
-        #    assert(False)
-        #except redis.ResponseError as e:
-        #    # Expecting an error.
-        #    assert("not defined" in str(e))
-        #    pass
+        try:
+           query = """CALL db.idx.fulltext.queryNodes('A', 'B') YIELD node AS n RETURN node"""
+           self.graph.query(query)
+           assert(False)
```

Ruff (0.15.10)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)